src/oeth/post-processors/daily-stats/daily-stats.ts

```diff
@@ -90,6 +109,7 @@ async function updateDailyStats(ctx: Context, date: Date) {
     lastRethRate,
     lastSfrxEthRate,
     lastWrappedOETHHistory,
+    holdersOverThreshold,
```

Comment:
maybe a better name is `holdersGte1e17`

amazing name I know

![pepe cringe](https://media.tenor.com/1Tfo0bJGWJMAAAAM/pepe.gif)

Reply:
it's different for OUSD and OETH so I kept it unspecific